breathecode/jobs/tests/mixins/jobs_test_case.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework.test import APITestCase
+from datetime import datetime, timedelta, date
+from breathecode.tests.mixins import GenerateModelsMixin, CacheMixin, TokenMixin, GenerateQueriesMixin, DatetimeMixin
+
+
+class JobsTestCase(APITestCase, GenerateModelsMixin, CacheMixin, TokenMixin, GenerateQueriesMixin,
+                   DatetimeMixin):
+    """CertificateTestCase with auth methods"""
+    def setUp(self):
+        self.generate_queries()
```

Comment:
If you do a pull from development you should implement the `BreathecodeMixin` and use the new mixin (`self.bc.help()`) https://github.com/jefer94/apiv2/blob/5aa5c119e86dd9ca118d7a1d51a24efb1f1412ed/breathecode/marketing/tests/mixins/marketing_test_case.py#L23